### DISCUSSION TITLE
Inconsistency of the font size for Callout component in Figma vs Paste site

### DISCUSSION THREAD

**Discussion by frankfzhao** [twilio-labs/paste#2662] [General] [answered]
Hey team,

Not sure if it's just inconsistency or sth else but I have discovered that for the Callout component, we are using 12x for the header and description (See screenshots attached)
<img width="1692" alt="image" src="https://user-images.githubusercontent.com/82524550/188709089-ff8dce28-7432-4334-89d7-7419de976d52.png">

And in the paste site, it actually uses h3 and regular p for the header and description which is 14px (See screenshots attached)
<img width="2083" alt="image" src="https://user-images.githubusercontent.com/82524550/188709280-6f676a91-f9ab-45df-b372-1d9d65fa2757.png">

I am wondering which size should I choose to use. thank you

**Comment by loreina** [comment, marked as answer]
Hi @frankfzhao, thanks for catching this. I made a branch to fix this in Figma and added you as a reviewer. Once that's reviewed, we'll merge and publish the changes. Thanks!

**Reply by serifluous** [reply]
(Conversation continued in Figma, but I'm replying here to close the loop). The fix has been published.
